src/OrcaSlicer.cpp

```diff
@@ -892,8 +892,6 @@ int CLI::run(int argc, char **argv)
     // instruct the window manager to fall back to X server mode.
     ::setenv("GDK_BACKEND", "x11", /* replace */ true);
 
-    ::setenv("WEBKIT_DISABLE_COMPOSITING_MODE", "1", /* replace */ false);
```

Comment:
It might break the web feature on some Linux destructions. Fedora IIRC

Reply:
With this setting present, it causes run away memory consumption and a segfault shortly after launch.

Reply:
I have also been getting segfaults on newer nightly builds of OrcaSlicer on Gentoo Linux.  I can confirm that removing this line fixes the segfault issue for me.